packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/webrtc/PeerConnectionManager.kt

```diff
@@ -400,7 +379,7 @@ internal class PeerConnectionManager(
     peerConnectionMutex.withLock {
       val pc = peerConnection!!
 
-      if (needsRestart && !isExWebrtc) {
+      if (needsRestart) {
         pc.restartIce()
       }
```

Comment:
**suggestion:** In web-sdk we removed this completely. We assume ice restarts are not needed during negotiation. In fact in web-sdk the restarts introduced failures. Try removing this and checking if everything works **OK**
```suggestion
```